Meadowlark-js/backends/meadowlark-postgresql-backend/test/setup/Config.ts

```diff
@@ -9,9 +9,17 @@ import { CachedEnvironmentConfigProvider, Config } from '@edfi/meadowlark-utilit
 import { join } from 'path';
 
 export const setupConfigForIntegration = async () => {
-  const path = join(process.cwd(), '.env');
+  // First look for .env file paired with this Config.ts file
+  let path = join(__dirname, '.env');
```

Comment:
allow for different .envs in each backend rather than force one big .env at root